src/conversions.jl

```diff
@@ -72,8 +73,16 @@ end
 # TAI <-> TT
 function rescale(::Type{TAIEpoch}, ep::TTEpoch)
     jd1, jd2 = julian1(ep), julian2(ep)
-    date, date1 = eraTttai(jd1, jd2)
-    TAIEpoch(date, date1)
+    dtat = OFFSET_TT_TAI/SECONDS_PER_DAY;
```

Comment:
The ported code should be in a separate function. Then you can also easily add a test for it.